template.go

```diff
@@ -149,6 +156,11 @@ func AddTemplateExt(ext string) {
 	beeTemplateExt = append(beeTemplateExt, ext)
 }
 
+func AddViewPath(viewPath string) error {
+	beeViewPathTemplates[viewPath] = make(map[string]*template.Template)
```

Comment:
As `AddViewPath` is  a public function, is it possible the user add the viewPath during the application running? That would cause map panic.